src/utils/fetch-twitch-videos.js

```diff
@@ -2,6 +2,20 @@ import { get } from './request';
 import { TWITCH_HEADERS, TWITCH_VIDEO_URL } from '../constants';
 
 // Fetches videos from twitch api
+
+const simplifyResponse = video => {
```

Comment:
Nice! This data shape consistency is definitely going to be helpful when working with other media types